### ISSUE TITLE
HTTPParser doesn't follow 302 redirect

### ISSUE THREAD

**Issue by hwbllmnn** [opengeospatial/teamengine#476]
**Describe the bug**

When the `HTTPParser` is used and the requested resource delivers a HTTP 302 redirect, it will not follow the redirect but deliver an empty body.

**To Reproduce**
1. Run any test with an URL which delivers a HTTP 302 redirect

**Expected behavior**

I would expect the `HTTPParser` to follow the `Location` header and deliver the response body of that URL.

**Screenshots**

not applicable

**Additional context**

none

**Comment by dstenger** [comment]
Thank you for reporting.

**Comment by g-weber** [comment]
Dear all, have you made any progress on this issue in the meantime?

**Comment by dstenger** [comment]
Following service can be used to reproduce the behavior: http://geoportal.bafg.de/arcgis1/rest/services/INSPIRE/Catchment/MapServer/exts/InspireView/service?VERSION=1.3.0&SERVICE=WMS&REQUEST=Getcapabilities

I assume that this error will only occur with CTL tests as TestNG tests use their own parsers.

**Comment by hwbllmnn** [comment]
True. As I wrote, I traced it to the  `HTTPParser` class which is only used from CTL AFAIK.

**Comment by dstenger** [comment]
We are planning to resolve this issue in Q4.
